src/App.js

```diff
@@ -25,7 +25,9 @@ const App = () => {
         </Route>
 
         <Route path="/add-view" component={AddView} />
-        <Route path="/list-view" component={ListView} />
+        <Route path="/list-view">
+          <ListView token={token} />
```

Comment:
@katia-utochkina should the token be passed in the add view instead?

Reply:
@ciradu2204, at the moment, I don't know (we don't have much happening in that component at the moment)

Reply:
Done!